packages/babel-jest/src/index.js

```diff
@@ -18,8 +18,8 @@ import type {
 import crypto from 'crypto';
 import fs from 'fs';
 import path from 'path';
+import {transform as babelTransform, util as babelUtil} from '@babel/core';
 import jestPreset from 'babel-preset-jest';
-import {transform as babelTransform, util as babelUtil} from 'babel-core';
```

Comment:
Is it possible for `process` to be async? Babel now exposes an async version of this function so that in the future we can start optionally allowing plugins some amount of async initialization. The more tooling that uses the async transforms, the better.

Reply:
In our case no, as we call it through `require` which must be synchronous